packages/firestore/test/integration/util/helpers.ts

```diff
@@ -86,6 +86,11 @@ export function toDataArray(
   return docSet.docs.map(d => d.data());
 }
 
+/** Converts a DocumentSet to an array with the id of each document */
+export function toIds(docSet: firestore.QuerySnapshot): String[] {
```

Comment:
There is a similar helper function on iOS and Android (I hope I didn't miss it here).

Reply:
It's possible it was omitted because map() is so easy in JS, but I'm not opposed to having it.